ChangeLog.md

```diff
@@ -8,8 +8,10 @@ and this project adheres to
 
 ### Added
 * Support for 2D systems in `freud.diffraction.StaticStructureFactorDebye`.
+* Mass dependence in `freud.cluster.ClusterProperties` and calculation of inertia tensor and center of mass.
```

Comment:
I am in the process of releasing version freud v2.11.0, so the changes made on this PR will be a part of the next release. That means that changelog entry for this feature will need a new section in the changelog. I can take care of this so the formatting is right.